sources/packages/api/src/app.module.ts

```diff
@@ -172,6 +174,8 @@ import { AppSupportingUsersModule } from "./app.supporting-users.module";
     InstitutionLocationControllerService,
     SFASApplicationService,
     SFASPartTimeApplicationsService,
+    GCNotifyService,
```

Comment:
Do we need this at the app level?

Reply:
It throwed an exception when removed, so added in the app level.

Reply:
Are all the modules where GCNotifyService is added a direct or transitive dependency of StudentFileService?

I see it is added to all the module.ts?

Reply:
yes